README.md

```diff
@@ -1,424 +1,159 @@
-- [Google Cloud Scheduler API documentation](https://cloud.google.com/scheduler/docs)
+To access a Google Cloud API, you will generally need to activate it in the
+cloud console. This typically involves three steps:
 
-### Secret Manager
+1.  If you have not created a Google Cloud Project, do so. Point your browser
+    to the Google Cloud Console at https://console.cloud.google.com, sign up if
+    needed, and create or choose a project. Make note of the project number
+    (which is numeric) or project ID (which is usually three or more words
+    separated by hyphens). Many services will require you to pass that
+    information in when calling an API.
 
-- `gem install google-cloud-secret_manager`
-- [google-cloud-secret_manager README](google-cloud-secret_manager/README.md)
-- [google-cloud-automl API documentation](https://googleapis.dev/ruby/google-cloud-secret_manager/latest)
-- [google-cloud-secret_manager on RubyGems](https://rubygems.org/gems/google-cloud-secret_manager/)
-- [Google Secret Manager documentation](https://cloud.google.com/secret-manager/docs)
+2.  For most services, you will need to provide billing information. If this is
+    your first time using Google Cloud Platform, you may be eligible for a free
+    trial.
 
-### Cloud Security Center
+3.  Enable the API you want to use. Click the "APIs & Services" tab in the left
+    navigation, then click the "Enable APIs and Services" button near the top.
+    Search for the API you want by name, and click "Enable". A few APIs may be
+    enabled automatically for you, but most APIs need to be enabled explicitly.
 
-- `gem install google-cloud-security_center`
-- [google-cloud-security_center README](google-cloud-security_center/README.md)
-- [google-cloud-security_center API documentation](https://googleapis.dev/ruby/google-cloud-security_center/latest)
-- [google-cloud-security_center on RubyGems](https://rubygems.org/gems/google-cloud-security_center)
-- [Google Cloud Security Center documentation](https://cloud.google.com/security-command-center/docs)
+Once you have a project set up and have enabled an API, you are ready to begin
+using a client library to call the API.
 
-### Cloud Spanner
+### Choosing a client library
 
-- `gem install google-cloud-spanner`
-- [google-cloud-spanner README](google-cloud-spanner/README.md)
-- [google-cloud-spanner API documentation](https://googleapis.dev/ruby/google-cloud-spanner/latest)
-- [google-cloud-spanner on RubyGems](https://rubygems.org/gems/google-cloud-spanner)
-- [Google Cloud Spanner documentation](https://cloud.google.com/spanner/docs)
+This repository contains two types of API client Rubygems: the _main library_
+for the API (e.g. the `google-cloud-translate` gem for the Translation service),
+and one ore more _versioned libraries_ for different versions of the service
+(e.g. `google-cloud-translate-v2` and `google-cloud-translate-v3` for versions
+2 and 3 of the service).
```

Comment:
I think it would be worthwhile to explain the difference between service versions and client library versions in a sentence or two, both here and in the Library Versioning section, below.

Reply:
Done